magpie/register.py

```diff
@@ -898,7 +916,7 @@ def _apply_profile(_usr_name=None, _grp_name=None):
                 from magpie.api.management.group.group_utils import create_group
                 return create_group(**grp_data)
 
-    def _validate_response(operation, is_create, item_type="Permission"):
+    def _validate_response(operation, is_create, item_type="Permission", raise_errors=False):
```

Comment:
Not needed to pass down the parameter since `_validate_response` is within `_apply_permission_entry`. 
Only parameters that change should be passed.